spouse/tf_model.py

```diff
@@ -2,56 +2,85 @@
 from typing import Tuple
 import numpy as np
 import pandas as pd
+from snorkel.types import DataPoint
+
 import tensorflow as tf
+from tensorflow.keras.layers import (
+    Bidirectional,
+    Concatenate,
+    Dense,
+    Embedding,
+    Input,
+    LSTM,
+)
+
+
```

Comment:
I thought there’s preprocessors for getting left and right tokens, any way to reuse them?

Reply:
Switched to using existing fields for right tokens and between tokens (which were already present). For left tokens, it's just one line, so I inlined it.